docs/quick-tutorials/k8s-kafka-mtls.mdx

```diff
@@ -309,59 +300,50 @@ kubectl logs -f --tail 1 -n otterize-tutorial-kafka-mtls deploy/client-authentic
 ```
 
 ```
-Loading mTLS certificates
-Connecting to Kafka
-Creating a producer for - transactions
-Sending messages
-Sent message - Message 1 [sent by client-authenticated]
-Sent message - Message 2 [sent by client-authenticated]
-Sent message - Message 3 [sent by client-authenticated]
+[client-authenticated] Loading mTLS certificates
+[client-authenticated] Connecting to Kafka
+[client-authenticated] Creating a producer and a consumer for - mytopic
+[client-authenticated] Sending messages
+[client-authenticated] Creating a producer and a consumer for - transactions
+[client-authenticated] Sending messages
+[debug: client-authenticated] Sent message to [mytopic] - [client-authenticated] Sent message 1
+[debug: client-authenticated] Sent message to [transactions] - [client-authenticated] Sent message 1
+
 ```
 
 </details>
 
-## Optional: Install Otterize CLI and examine topic-level Kafka access map
+## Require authentication
 
-Now that our 3 clients are accessing our Kafka server, which has the Otterize OSS Kafka watcher enabled and feeding data to the network mapper, we can query the network mapper directly
-to see the map it has built of topic-level access by those clients.
+This is our first step in protecting our Kafka service. Since we have configured the client processes to generate and mount an mTLS certificate we can safely turn on the `clientIdentityRequired: true` flag in our `kafkaserverconfig.yaml` file.
 
 <details>
-<summary>Install the Otterize CLI</summary>
+<summary>Turn on tLS authentication</summary>
```

Comment:
```suggestion
<summary>See changes needed to enable clientIdentityRequired</summary>
```
I'd also remove the collapsible so it just shows, and make it highlight the changed lines for clarity.